src/Platforms/DB2Platform.php

```diff
@@ -610,10 +610,6 @@ public function getAlterTableSQL(TableDiff $diff)
                     $columnDiff->column->getQuotedName($this),
                     $this->getColumnComment($columnDiff->column)
                 );
-
-                if (count($columnDiff->changedProperties) === 1) {
```

Comment:
The real reason for this change is to get rid of direct access to `ColumnDiff::$changedProperties` and enable further refactoring of the schema comparison API.

https://github.com/doctrine/dbal/pull/5620 was implemented for the same purpose as well. More changes are to follow.